todo_app/templates/index.html

```diff
@@ -2,17 +2,149 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
-  <div class="jumbotron">
+  <div class="jumbotron jumbotron-fluid">
     <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+    <p class="lead">Leon's to-do app 'Today, Ta-Da!'.</p>
+    <input type="button" onclick="alert('v1.1 beta')" value="ABOUT">
   </div>
 
+  <div class="row">
+    <form action="/add-item" method="POST">
+      <div class="form">
+        <input type="submit" value="Add todo item">
+        <br>
+        <label for="title">Todo item title: </label>
+        <input type="text" name="title" id="title" required>
+      </div>
+  </div>  
+  <br><br>
+  </form>
+<!--
+  <div class="row">
+    <form action="/del-item" method="POST">
+      <div class="form">
+        <input type="submit" value="Delete todo item">
+        <label for="title">Todo item id: </label>
+        <input type="number" name="del_id" id="del_id" >
+      </div>
+  </div>  
+  <br><br>
+-->
```

Comment:
I assume this is just commented out as the delete functionality wasn't quite working - see my comment in `session_items.py` for that and then we may be able to switch this one 🙂